docs/reference/github-tables.md

```diff
@@ -247,6 +247,28 @@ SELECT * FROM github_repo_prs('mergestat/mergestat');
 SELECT * FROM github_repo_prs('mergestat', 'mergestat'); -- both are equivalent
 ```
 
+## `github_repo_branches`
+
+Table-valued-function that returns branch information from a GitHub repository.
+
+| Column                           | Type     |
+|----------------------------------|----------|
+| owner                            | TEXT     |
+| repo_name                        | TEXT     |
```

Comment:
```suggestion
```

Reply:
let's remove these - since they're "hidden" columns and won't appear to the user